### ISSUE TITLE
need help with enabling flatpak permissions.

### ISSUE THREAD

**Issue by DOOMMARINE117** [tchx84/Flatseal#557]
i just installed flatseal and flatpak for archlinux. cause im using bottles to install games and etc.

im assuming i need permissions. what command to get the permissions?

it shows no applications found.

**Comment by rusty-snake** [comment]
Please use a better title. Even if it is a common error, there are 10 different common errors.


Did you looked at https://github.com/tchx84/Flatseal/issues?q=is%3Aissue+no+applications+found+is%3Aclosed ?

How did you installed flatseal? Via flatpak from flathub or AUR?

Do you have any global overrides for flatpak permissions?

Do you use the default installation or a custom installation?

Which other flatpaks do you have installed?

**Comment by DOOMMARINE117** [comment]
> Please use a better title. Even if it is a common error, there are 10 different common errors.
> 
> Did you looked at https://github.com/tchx84/Flatseal/issues?q=is%3Aissue+no+applications+found+is%3Aclosed ?
> 
> How did you installed flatseal? Via flatpak from flathub or AUR?
> 
> Do you have any global overrides for flatpak permissions?
> 
> Do you use the default installation or a custom installation?
> 
> Which other flatpaks do you have installed?

i installed it via terminal or the add/software removal program. 

i looked for other issues to see if any had similar. nothing so far. 

global overides? no i dont think so. how to enable them?

im assuming default one. 

Also, i tried using this command. flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo

it gives me this: 

'/var/lib/flatpak/exports/share'
'/home/talion/.local/share/flatpak/exports/share'

are not in the search path set by the XDG_DATA_DIRS environment variable, so
applications installed by Flatpak may not appear on your desktop until the
session is restarted.

**Comment by rusty-snake** [comment]
> i installed it via terminal or the add/software removal program.

or?

> Which other flatpaks do you have installed?

What does `flatpak list` show?

**Comment by DOOMMARINE117** [comment]
> > i installed it via terminal or the add/software removal program.
> 
> or?
> 
> > Which other flatpaks do you have installed?
> 
> What does `flatpak list` show?

i tried it. it showed nothing.

**Comment by rusty-snake** [comment]
So you don't have any flatpaks installed. And flatseal says 'no applications found'. See no problem.

**Comment by DOOMMARINE117** [comment]
> So you don't have any flatpaks installed. And flatseal says 'no applications found'. See no problem.

gotcha. 

im curious. i was trying to install a flathub program and go through add/removal software. but it doesnt load to the program in the software manager.

is it suppose to do that?

**Comment by rusty-snake** [comment]
Can you rephrase, I don't get what you mean.

Also what do you mean by "software manager"?

**Comment by DOOMMARINE117** [comment]
> Can you rephrase, I don't get what you mean.
> 
> Also what do you mean by "software manager"?

![IMG_20230815_114551](https://github.com/tchx84/Flatseal/assets/64376960/dcd6595e-5418-4692-ac21-79f1f2776a07)

**Comment by DOOMMARINE117** [comment]
in arch linux, it has a different software manager. idk what the name is. 

when  you select the exe from flathub of flatseal it wont load through the manager.

**Comment by DOOMMARINE117** [comment]
![Uploading IMG_20230815_121954.jpg…]()

**Comment by DOOMMARINE117** [comment]
![IMG_20230815_121954](https://github.com/tchx84/Flatseal/assets/64376960/a9c358b4-1045-43cf-b9ae-6a0b7590232a)

**Comment by rusty-snake** [comment]
>  idk what the name is.

3 dots top right > about software

> [IMAGE]

Generally, posting images is bad unless you want to show graphics (accessibility, bandwidth, cli/tui clients, ...) and if you need screenshots have better quality (angel, ...) than pictures of the screen.

---

To summarize, you problem is that ... you can not install flatpaks (from flathub)? If so, try it on the command-line. If that's work you may want to report it to your GUI software manager or ask in the arch forum.

**Comment by DOOMMARINE117** [comment]
> 



> > idk what the name is.
> 
> 3 dots top right > about software
> 
> > [IMAGE]
> 
> Generally, posting images is bad unless you want to show graphics (accessibility, bandwidth, cli/tui clients, ...) and if you need screenshots have better quality (angel, ...) than pictures of the screen.
> 
> To summarize, you problem is that ... you can not install flatpaks (from flathub)? If so, try it on the command-line. If that's work you may want to report it to your GUI software manager or ask in the arch forum.

i see, so posting pictures is not recommended. i have a disability. so explaining is always difficult. but, i do my best to explain. 

yeah that would be the issue of flathub. its interesting. in linux mint. its already enabled and that worked. so there maybe i need besides flatpak. 

what command-line do i need? i look on the about. its pamac

**Comment by rusty-snake** [comment]
Make sure `flatpak remotes` shows `flathub`. Install a program from flathub like `flatpak install flathub org.gnome.TextEditor`.

Docs:

- https://wiki.archlinux.org/title/Flatpak
- https://flathub.org/setup
- https://docs.flatpak.org/en/latest/

**Comment by DOOMMARINE117** [comment]
> Make sure `flatpak remotes` shows `flathub`. Install a program from flathub like `flatpak install flathub org.gnome.TextEditor`.
> 
> Docs:
> 
> * https://wiki.archlinux.org/title/Flatpak
> * https://flathub.org/setup
> * https://docs.flatpak.org/en/latest/

so i type in flathub remotes.

flathub system
name    system

i installed the flatpak install flathub org.gnome.TextEditor what's next?

**Comment by DOOMMARINE117** [comment]
ok the program works now. 

thank you for the help.

**Comment by DOOMMARINE117** [comment]
also, before i close this.

i can use flatseal for gaming right after i add bottles in?

i use dvd games, epic games and gog galaxy.

**Comment by rusty-snake** [comment]
> i can use flatseal for gaming

No.

You can, however, change permissions of games installed as flatpaks or in flatpaks (then you change the permissions of the flatpak like bottles or steam).

**Comment by DOOMMARINE117** [comment]
> > i can use flatseal for gaming
> 
> No.
> 
> You can, however, change permissions of games installed as flatpaks or in flatpaks (then you change the permissions of the flatpak like bottles or steam).

how do i put them as flatpaks?

**Comment by rusty-snake** [comment]
If you want to move existing data, ask at a place where people use bottles / have knowledge.

Otherwise https://flathub.org/apps/com.usebottles.bottles

**Comment by DOOMMARINE117** [comment]
> If you want to move existing data, ask at a place where people use bottles / have knowledge.
> 
> Otherwise https://flathub.org/apps/com.usebottles.bottles

cool, thank you for your help.

that be it for now.
